duolingo.py

```diff
@@ -35,10 +38,22 @@ def _login(self):
         raise Exception("Login failed")
 
     def get_activity_stream(self, before=None):
+        """
+        Get user's activity stream from
+        ``https://www.duolingo.com/stream/<user_id>?before=<date> if before
+        date is given or else
+        ``https://www.duolingo.com/activity/<user_id>``
+
+        :param before: Datetime in format '2015-07-06 05:42:24'
+        :type before: str
+        :rtype: dict
+        """
         if before:
-            url = "https://www.duolingo.com/stream/{}?before={}".format(self.user_data.id, before)
+            url = "https://www.duolingo.com/stream/{}?before={}".format(
+                self.user_data.id, before)
```

Comment:
hey @ZuluPro I know you are trying to follow PEP8 but I think some readability things that might not necessarily support PEP8 are okay to implement.

For this `format()` block along with a few more below, it might be best to align the new line with the beginning of the `"http:...` instead of `=`